build.gradle

```diff
@@ -29,42 +17,20 @@ plugins {
 repositories {
     mavenLocal()
     mavenCentral()
-//    maven {
-//        url = 'https://jitpack.io'
-//    }
 }
 
 // Rails variables
-//group = 'net.sf'
-version = '2.2-alpha'
-description = 'Rails (18xx)'
```

Comment:
This is a gradle property so we should add it back (and maybe use it instead of the `railsDescription` we have below?

Reply:
nods at JKlap : yes